--- contact.html.var	2020-12-22 23:00:50.989102230 +0100
+++ contact.html.var.new	2020-12-22 23:02:43.043587625 +0100
@@ -77,7 +77,7 @@
 Content-language: nb
 Content-type: text/html; charset=UTF-8
 Body:----------nb--
-Om du tror dette skyldes en serverfeil, venligst kontakt
+Om du tror dette skyldes en tjenerfeil, venligst kontakt
 <a href="mailto:<!--#echo encoding="url" var="SERVER_ADMIN" -->">webansvarlig</a>.
 ----------nb--
 
